src/contracts/GPv2Settlement.sol

```diff
@@ -100,24 +106,128 @@ contract GPv2Settlement {
     /// @param clearingPrices An array of clearing prices where the `i`-th price
     /// is for the `i`-th token in the [`tokens`] array.
     /// @param feeFactor The fee factor to use for the trade.
-    /// @param encodedOrders Encoded signed EOA orders.
+    /// @param encodedTrades Encoded trades for signed EOA orders.
     /// @param encodedInteractions Encoded smart contract interactions.
     /// @param encodedOrderRefunds Encoded order refunds for clearing storage
     /// related to invalid orders.
     function settle(
         IERC20[] calldata tokens,
         uint256[] calldata clearingPrices,
         uint256 feeFactor,
-        bytes calldata encodedOrders,
+        bytes calldata encodedTrades,
         bytes calldata encodedInteractions,
         bytes calldata encodedOrderRefunds
     ) external view onlySolver {
         require(tokens.length == 0, "not yet implemented");
         require(clearingPrices.length == 0, "not yet implemented");
         require(feeFactor == 0, "not yet implemented");
-        require(encodedOrders.length == 0, "not yet implemented");
+        require(encodedTrades.length == 0, "not yet implemented");
         require(encodedInteractions.length == 0, "not yet implemented");
         require(encodedOrderRefunds.length == 0, "not yet implemented");
         revert("Final: not yet implemented");
     }
+
+    /// @dev Process all trades for EOA orders one at a time returning the
+    /// computed net in and out transfers for the trades.
+    ///
+    /// This method reverts if processing of any single trade fails. See
+    /// [`processOrder`] for more details.
+    /// @param tokens An array of ERC20 tokens to be traded in the settlement.
+    /// @param clearingPrices An array of token clearing prices.
+    /// @param encodedTrades Encoded trades for signed EOA orders.
+    /// @return inTransfers Array of transfers into the settlement contract.
+    /// @return outTransfers Array of transfers to pay out to EOAs.
+    function processTrades(
```

Comment:
nit: not sure if "processing" is the right word here, as we are not actually executing any trades. Maybe parse is a more precise word?

Reply:
I'm not 100% sold on `parse` either as we aren't really parsing the trade but rather computing the transfers required to execute the trade. Eventually we will also:
- verify order protection (like limit price is respected)
- verify order not being replayed

Which is why I used "process". But I agree, its not super descriptive either.

Reply:
Maybe `computeTradeExecution`?